azure-pipelines.yml

```diff
@@ -0,0 +1,174 @@
+variables:
+  # The $(FlutterToolPath) variable is generated by the FlutterInstall task 
+  # See code in the Flutter Build task https://github.com/aloisdeniel/vsts-flutter-tasks/blob/master/tasks/build/index.ts
+  flutterExecPath: $(FlutterToolPath)/flutter.bat
+  versionNumberRegex: '(?<=version: ).*'
+
+trigger:
+  batch: 'true'
+  branches:
+    include:
+      - develop
+      - release
+      - hotfix
+      - master
+  paths:
+    include:
+      - device_calendar/*
+
+stages:
+  # ----------- CI -----------
+  - stage: Build
+    jobs: 
+     - job: BuildAndroidAndIos
+       pool:
+        vmImage: 'macOS-latest'
+       steps:
+        - task: FlutterInstall@0
+          displayName: 'Flutter install'
+          inputs:
+            channel: 'stable'
+            version: 'latest'
+
+        - task: FlutterBuild@0
+          displayName: 'Flutter build - Android'
+          inputs:
+            target: 'aab'
+            projectDirectory: 'device_calendar/example'
+
+        - task: FlutterBuild@0
+          displayName: 'Flutter build - iOS'
+          inputs:
+            target: 'ios'
+            projectDirectory: 'device_calendar/example'
+            iosCodesign: false
+            
+  # -----------CD Pre-release -----------
+  - stage: DevelopmentRelease
+    dependsOn: Build
+    condition: and(succeeded(), in(variables['Build.SourceBranch'], 'refs/heads/release', 'refs/heads/hotfix'))
+    jobs:
+     - job: ReleaseDevelopmentVersion
+       pool:
+        vmImage: 'windows-latest'
+       steps:
+        - task: FlutterInstall@0
+          displayName: 'Flutter install'
+          inputs:
+            channel: 'stable'
+            version: 'latest'
+
+        - task: PowerShell@2
+          condition: eq(variables['Build.SourceBranch'], 'refs/heads/release')
+          displayName: 'Pre-release versioning - pubspec.yaml'
+          inputs:
+            targetType: 'inline'
+            script: |
+              [string] $version = select-string -path "$(Build.SourcesDirectory)/device_calendar/pubspec.yaml" -pattern "$(versionNumberRegex)" | %{ $_.Matches[0].Value }
+              Write-Host "##vso[task.setvariable variable=currentVersion]$version"
+
+              [string] $pubspecContent = Get-Content -Path "$(Build.SourcesDirectory)/device_calendar/pubspec.yaml" -Raw
+
+              Write-Host " (i) Current version: $version"
+              Write-Host " (i) Original pubspec.yaml content: $pubspecContent"
+
+              [bool] $hasMatches = $pubspecContent -match $versionRegex
+              
+              [string] $newPubspecContent = $pubspecContent -replace $version, "$version-dev.$(Build.BuildID)"
+
+              Write-Host " (i) Updated pubspec.yaml content: $newPubspecContent"
+
+              $newPubspecContent | Set-Content -Path "$(Build.SourcesDirectory)/device_calendar/pubspec.yaml"
+
+        - task: PowerShell@2
+          condition: eq(variables['Build.SourceBranch'], 'refs/heads/release')
```

Comment:
and remove this condition

Reply:
Seems like you're looking at the old commit, I've removed them already